packages/lib/src/components/internal/PersonalDetails/PersonalDetails.tsx

```diff
@@ -82,7 +82,7 @@ export default function PersonalDetails(props: PersonalDetailsProps) {
             )}
 
             {requiredFields.includes('gender') && (
-                <Field errorMessage={!!errors.gender} classNameModifiers={['gender']} name={'gender'}>
+                <Field errorMessage={!!errors.gender} classNameModifiers={['gender']} name={'gender'} useLabelElement={false}>
                     {renderFormField('radio', {
                         i18n,
```

Comment:
Wouldn't be better to analyze the component child at the `<Field>` component implementation, and if it is a Radio element, then we decide there how to proceed (label or div)?

Something like:
```js
const isRadio = toChildArray(children).some((child: ComponentChild) => child.type === RadioGroup);
```
Then we do not need to always set the `useLabelElement` prop when using `radio` input. Having to always do this can lead to inconsistency as we can forget to set this property.

Reply:
> ...we can forget to set this property

You could say this is true of _any_ property.

Also then `Field` needs to know of the existence of a thing called `RadioGroup` (and any other component types that need a `div` rather than a `label`. KYC, for example, has other such components).

I do see what you're saying however - I'll bear it in mind, I think what you suggest might be something that can be implemented in the "shared-components" version of this Field comp (which has a project specific wrapper around it, where knowledge of things like `RadioGroup` might make more sense